src/applications/public-outreach-materials/libraries/library-card.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * Creates cards for asset library.
+ */
+export function libraryCard(jsonData) {
+  let output = '';
+  let url;
+
+  function fileType(data) {
+    const split = data.split('.');
+    return split.slice(-1).pop();
+  }
+
+  jsonData.forEach((json, index) => {
+    if (json.fieldMedia) {
+      const type = json.fieldMedia.entity.entityBundle;
+      let assetText;
+      let format;
+      switch (type) {
+        case 'document':
+          url = json.fieldMedia.entity.fieldDocument.entity.url;
+          if (Object.hasOwnProperty.call(json, 'derivedFields')) {
+            url = json.derivedFields.absoluteUrl;
+          }
+          format = fileType(url);
+          assetText = `<a class="file-download-with-icon" target="_blank" href="${url}" download="${url}"><i class="fas fa-download vads-u-padding-right--1"></i>Download ${format.toUpperCase()}</a>`;
```

Comment:
Are the urls here pointing to an AWS bucket? If so, we probably want to add `rel="noopener noreferrer"`